frontend/pages/media/marketing.js

```diff
@@ -42,37 +45,34 @@ const MarketingContent = () => {
     setCurrentPage("All");
   };
 
-  // scroll to top on load
-  useEffect(() => window.scrollTo(0, 0), []);
-
   // get marketing archives
   // input: marketing archives data from google sheets
   // output: array of dictionaries containing marketing archives data
+  const fetchMarketingArchives = async () => {
+    const res = await getMarketingArchives();
+    const tempContent = humps.camelizeKeys(res);
```

Comment:
Is there a reason why the year filtering methods (e.g. reverse() and filter() were removed for `tempContent`? Seems to be causing issues with the timeline